### ISSUE TITLE
promptForPushPermission not working.

### ISSUE THREAD

**Issue by rahulnainwalttn** [CleverTap/clevertap-react-native#383]
promptForPushPermission not working in android.

```
CleverTap.isPushPermissionGranted((err, res) => {
    // console.log('isPushPermissionGranted', res, err);
    if (!res) {
      CleverTap.promptForPushPermission(true);
    }
  });
```

**Comment by Anush-Shand** [comment]
@rahulnainwalttn This feature is only available for Android 13 and above devices as mentioned in these docs https://developer.clevertap.com/docs/react-native-push-notification#push-primer

This works fine on our demo app. If this issue exists on an Android 13 or higher device on your end then please share a reproducible example and verbose logs for us to investigate further

**Comment by rahulnainwalttn** [comment]
@Anush-Shand Yes I am facing it in Android 14.
Will share a reproducible example.

**Comment by Anush-Shand** [comment]
@rahulnainwalttn Was this problem resolved on your end? We can close this issue, if so

**Comment by rahulnainwalttn** [comment]
@Anush-Shand No, after that I didn't work on it. Will share a reproducible example once I resume work on it.

**Comment by Anush-Shand** [comment]
@rahulnainwalttn Is there any update on this?

**Comment by Anush-Shand** [comment]
Closing this due to inactivity. Feel free to reopen @rahulnainwalttn
